### PR TITLE
Created Footer

### PR DESCRIPTION
**Pull request by irinayarmolenko84**
Created footer

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,12 +2,59 @@
...
+            </h3>
+            <h3>
+                <a href="#">Поповнення мобільного</a>
+            </h3>
+            <h3>
+                <a href="#">Контакти</a>
+            </h3>
+            <h3>
+                <a href="#">Мобільні додатки</a>
+            </h3>
```
Comment (SVladikO) [inline src/index.html:29]: without h3

File: src/style/layout/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
+footer {
+  background-color: $color_text_fill_9;
+  color: $color_bg_white;
+  font-family: Arial;
```
Comment (SVladikO) [inline src/style/layout/_footer.scss:4]: where did you get this font?

File: src/style/layout/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  .container {
+    grid-template-columns: 1fr;
+  }
+}
+
+h4 {
+  opacity: 50%;
+}
+
+h3 a {
```
Comment (SVladikO) [inline src/style/layout/_footer.scss:26]: without h3

File: src/style/layout/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  text-decoration: none;
+  cursor: pointer;
+  color: white;
```
Comment (SVladikO) [inline src/style/layout/_footer.scss:29]: we have color variables for these porposes

File: src/index.html [hunk 5]
```diff
@@ -2,12 +2,59 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
-  <title>Oschadbank</title>
+    <meta charset="UTF-8">
+    <title>Oschadbank</title>
+    <link rel="stylesheet" media="screen" href="">
```
Comment (SVladikO) [inline src/index.html:7]: Hi. Please use this this code
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`
instead

File: src/index.html (hunk 1 above)
Comment (irinayarmolenko84) [reply src/index.html]: Done

File: src/style/layout/_footer.scss (hunk 3 above)
Comment (irinayarmolenko84) [reply src/style/layout/_footer.scss]: Done

File: src/style/layout/_footer.scss (hunk 4 above)
Comment (irinayarmolenko84) [reply src/style/layout/_footer.scss]: Done

File: src/index.html (hunk 5 above)
Comment (irinayarmolenko84) [reply src/index.html]: Done

**Review by SVladikO — APPROVED** [review]
Well done

**Pull request merged**